test/suites/ubuntu-core-edgex-image/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+sudo apt-get update
+sudo apt install qemu-kvm
+sudo apt install ovmf
```

Comment:
Please move to readme. Add the `kvm-ok` command necessary to test the environment. Link to https://ubuntu.com/core/docs/testing-with-qemu and other references.

Reply:
moved.